tools/stack_decode.py

```diff
@@ -116,7 +131,7 @@ def ignore_decoding_errors(io_wrapper):
 
 if __name__ == "__main__":
     if len(sys.argv) > 2 and sys.argv[1] == '-s':
-        decode_stacktrace_log(sys.argv[2], ignore_decoding_errors(sys.stdin))
+        decode_stacktrace_log(sys.argv[2], ignore_decoding_errors(sys.stdin), None)
```

Comment:
To avoid having two ways of doing things, should we remove the  `offset = find_address_offset(rununder.pid)` when we using subprocess.Popen a few lines below. This means we don't need to depend on pmap for the other usecase.